examples/express/.env.example

```diff
@@ -1,4 +1,4 @@
 SPLUNK_ACCESS_TOKEN='0123456789abcdefghijkl'
 OTEL_SERVICE_NAME='collectorless-express-example'
-OTEL_EXPORTER_JAEGER_ENDPOINT='https://ingest.us0.signalfx.com/v2/trace'
+OTEL_EXPORTER_OTLP_ENDPOINT='https://ingest.us0.signalfx.com/v2/trace/otlp'
```

Comment:
Do we use OTLP HTTP or gRPC exporter now and is it supported by SFX ingest?

Reply:
We are using OTLP/HTTP which is supported by the endpoint listed :point_up: